src/modules/real-time-metrics/filters/load-dataset-available-filters.js

```diff
@@ -67,22 +65,22 @@ const verifyWhitelistFields = ({ name, type: { name: typeName } }) => {
  * @return {Promise<Array<Object>>} An array of objects representing the available filters.
  */
 export default async function LoadDatasetAvailableFilters(dataset) {
-  if (cancelRequest) cancelRequest.cancel()
-  const ReportsRequestToken = Axios.CancelToken
-  cancelRequest = ReportsRequestToken.source()
+  if (abortController) abortController.abort()
+  abortController = new AbortController()
 
   const datasetsQuery = inputListByDatasetQuery(dataset)
 
   const graphqlQuery = {
     query: datasetsQuery
   }
 
-  const beholderService = new BeholderService({ cancelRequest })
-
   let data = []
 
   try {
-    const type = await beholderService.gql(graphqlQuery)
+    const type = await loadRealTimeMetricsData({
```

Comment:
renaming of the  variable to avoid no context in line 84 ( type.__type). I also suggest to rename __type, to match project standard

Reply:
Isso vem da API. Por ser chamada graphql, não tem como fazer um adapter pra esse cenário